Magento2/Sniffs/Legacy/WidgetXMLSniff.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
```

Comment:
We should use the Adobe copyright instead

Reply:
All PR has the Magento copyright... Don't know what is the more appropiate.

Reply:
actually it's for Magento in this case